src/pages/restaurant/components/menu/helpers/getUpdatedQuantityPerDishById.js

```diff
@@ -0,0 +1,21 @@
+//Helpers
+import _omit from 'lodash/omit';
+
+function getUpdatedQuantityPerDishById(
```

Comment:
you're directly manipulating the object that has been passed in the parameter, check this fn once